CHANGELOG.md

```diff
@@ -5,7 +5,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
-- Upgrade to GLFW 3.3
+- Pending via #211: Upgrade to GLFW 3.3
```

Comment:
I took the freedom to add this prefix because that change is not yet actually merged into master. Once the MR is through you should remember to remove the prefix here again. Normally I would ask authors of MRs to update the CHANGELOG on their branch so the changes only show up on the changelog of master once the change actually made it into that branch.